hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/checksum/ContainerMerkleTreeTestUtils.java

```diff
@@ -87,8 +91,8 @@ public static void assertTreesSortedAndMatch(ContainerProtos.ContainerMerkleTree
    *     "bytesPerChecksum" amount of data and are assumed to be contiguous.
    * @return The ChunkInfo proto object built from this information.
    */
-  public static ChunkInfo buildChunk(ConfigurationSource config, int indexInBlock, ByteBuffer... chunkChecksums)
-      throws IOException {
+  public static ContainerProtos.ChunkInfo buildChunk(ConfigurationSource config, int indexInBlock,
```

Comment:
Add the import explicitly to avoid having to type `ContainerProtos.ChunkInfo`

Reply:
I had been doing it with `ContainerProtos.ChunkInfo` too since there is the other existing Java object called `ChunkInfo`. IDE will differentiate them though, so it's a toss up which one is more readable.

Reply:
Yes I agree. It's hard to differentiate between `ChunkInfo` and `ContainerProtos.ChunkInfo` unless we inspect it.

Reply:
Ok, we can keep it.